src/index.js

```diff
@@ -0,0 +1,171 @@
+  newAdd.descreption = document.getElementById('insert').value;
+  newAdd.completed = false;
+  list.push(newAdd);
+  update();
+}
+
+function pushContent() {
+  removeChildNodes(listItems);
+  insert.value = '';
+  // insert.focus();
+  list.forEach((element) => {
+    const listItem = document.createElement('li');
+    const check = document.createElement('div');
+    check.className = 'check';
+    const input = document.createElement('input');
+    input.type = 'checkbox';
+    input.className = 'checkbox';
+    const descreForm = document.createElement('form');
+    descreForm.className = 'desc-form';
+    const descreption = document.createElement('input');
+    descreption.type = 'text';
+    descreption.className = 'descreption';
+    descreption.value = element.descreption;
+    const icon1 = document.createElement('button');
+    icon1.innerHTML = '<i class="fas fa-ellipsis-v"></i>';
+    icon1.className = 'fas fa-ellipsis-v';
+    const icon2 = document.createElement('button');
+    icon2.className = 'fa fa-trash';
+    icon2.innerHTML = '<i class="fas fa-trash">u</i>';
+    descreForm.appendChild(descreption);
+    check.appendChild(input);
+    check.appendChild(descreForm);
+    listItem.appendChild(check);
+    listItem.appendChild(icon1);
+    listItem.appendChild(icon2);
+    listItems.appendChild(listItem);
+  });
+  update();
+  refreshStorage();
+}
+
+function removeToDO() {
+  const remove = Array.from(document.querySelectorAll('.fa-trash'));
+  for (let i = 0; i < list.length; i += 1) {
+    remove[i].addEventListener('click', () => {
+      list.splice(i, 1);
+      listItems.removeChild(listItems.childNodes[i]);
+      window.location.reload();
+      update();
+      refreshStorage();
+    });
+  }
+}
+
+function checked() {
+  const checks = Array.from(document.querySelectorAll('.checkbox'));
+  const descreption = Array.from(document.querySelectorAll('.descreption'));
+  for (let i = 0; i < list.length; i += 1) {
+    if (list[i].completed) {
+      descreption[i].classList.toggle('checked');
+      checks[i].toggleAttribute('checked');
+      refreshStorage();
+    }
+  }
+}
+
+function checkBox() {
+  const checks = Array.from(document.querySelectorAll('.checkbox'));
+  const descreption = Array.from(document.querySelectorAll('.descreption'));
+  for (let i = 0; i < list.length; i += 1) {
+    checks[i].addEventListener('click', () => {
+      list[i].completed = checks[i].checked;
+      descreption[i].classList.toggle('checked');
+      checks[i].toggleAttribute('checked');
+      refreshStorage();
+    });
+  }
+}
+
+function edit() {
+  const descForm = Array.from(document.querySelectorAll('.desc-form'));
+  const enterDes = Array.from(document.querySelectorAll('.descreption'));
+  for (let i = 0; i < list.length; i += 1) {
+    descForm[i].addEventListener('submit', (e) => {
+      e.preventDefault();
+      list[i].descreption = enterDes[i].value;
+      refreshStorage();
+    });
+  }
+}
```

Comment:
- [ ] When I edit Task1 it worked, but afterward, when I try to edit Task3 it works on the DOM but failed to be reflected on the local storage. Kindly fix it! :pray: 

![Screenshot from 2021-10-29 22-23-15](https://user-images.githubusercontent.com/64894927/139491085-17a89cf8-4e01-4b79-b824-50208d2f2333.png)